test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -568,26 +568,31 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 				}, util.Timeout, util.Interval).Should(gomega.Succeed())
 			})
 
-			createdRequest := &autoscaling.ProvisioningRequest{}
 			provReqKey := types.NamespacedName{
 				Namespace: wlKey.Namespace,
 				Name:      provisioning.GetProvisioningRequestName(wlKey.Name, ac.Name, 1),
 			}
-
+			createdRequest := &autoscaling.ProvisioningRequest{
+				ObjectMeta: metav1.ObjectMeta{
```

Comment:
Do we need to provide the `ObjectMeta` here? I think it will be populated on the `Get`?

Reply:
Sorry I'm still greenhand on this. Really appreciate for your patience.